migrations/cockroach-secure/V1.1__eventstore.sql

```diff
@@ -27,3 +28,4 @@ CREATE TABLE eventstore.events (
     INDEX agg_type_agg_id (aggregate_type, aggregate_id),
     CONSTRAINT previous_sequence_unique UNIQUE (previous_sequence DESC)
 );
+ALTER SEQUENCE eventstore.event_seq OWNED BY eventstore.events.event_sequence;
```

Comment:
@adlerhurst have you added this line when it was in the orbos repo?

Reply:
nope ive never used `owned by`